cardano-testnet/test/Test/ShutdownOnSlotSynced.hs

```diff
@@ -57,15 +58,24 @@ hprop_shutdownOnSlotSynced = H.integrationRetryWorkspace 2 "shutdown-on-slot-syn
     H.cat (nodeStdout node)
     H.cat (nodeStderr node)
   mExitCodeRunning === Right ExitSuccess
+
   logs <- H.readFile (nodeStdout node)
-  slotTip <- case find (isInfixOf "Closed db with immutable tip") (reverse (lines logs)) of
-    Nothing -> H.failMessage callStack "Could not find current tip in node's log."
```

Comment:
The string `"Could not find current tip in node's log."` is probably a copy-paste error.  I think it should say `"Could not find close DB message"`